starport/interface/cli/starport/cmd/network_chain_join.go

```diff
@@ -92,7 +78,7 @@ func networkChainJoinHandler(cmd *cobra.Command, args []string) error {
 	if err := cliquiz.Ask(
 		cliquiz.NewQuestion("Account coins",
 			&account.Coins,
-			cliquiz.DefaultAnswer(strings.Join(acc.Coins, ",")),
+			cliquiz.DefaultAnswer("1000token,100000000stake"),
```

Comment:
I think it should be still good to user the value from `config` if it exists. Users may enter the default values for simplicity but it makes some troubles if they are not the same in the `config`

Reply:
solved by a9e99c7.